src/interventions/chrome-ios-62-63.js

```diff
@@ -0,0 +1,147 @@
+ * instances of PaymentItem, if any.
+ * @param {(Array<!window.PaymentItem>|undefined)} displayItems
+ */
+function convertDisplayItems(displayItems) {
+  if (!displayItems || !(displayItems instanceof Array)) {
+    return;
+  }
+
+  for (let i = 0; i < displayItems.length; i++) {
+    convertPaymentCurrencyAmount(displayItems[i].amount);
+  }
+}
+
+/**
+ * Converts PaymentCurrencyAmount.value instances from number to string inside
+ * instances of PaymentShippingOption, if any.
+ * @param {(Array<!window.PaymentShippingOption>|undefined)} shippingOptions
+ */
+function convertShippingOptions(shippingOptions) {
+  if (!shippingOptions || !(shippingOptions instanceof Array)) {
+    return;
+  }
+
+  for (let i = 0; i < shippingOptions.length; i++) {
+    convertPaymentCurrencyAmount(shippingOptions[i].amount);
+  }
+}
+
+/**
+ * Converts PaymentCurrencyAmount.value instances from number to string inside
+ * an instance of PaymentDetails, if any.
+ * @param {(window.PaymentDetails|undefined)} details
+ */
+function convertPaymentDetails(details) {
+  if (!details) {
+    return;
+  }
+
+  if (details.total) {
+    convertPaymentCurrencyAmount(details.total.amount);
+  }
+  convertDisplayItems(details.displayItems);
+  convertShippingOptions(details.shippingOptions);
+  convertPaymentDetailsModifiers(details.modifiers);
+}
+
+/**
+ * Converts PaymentCurrencyAmount.value instances from number to string inside
+ * instances of PaymentDetailsModifier, if any.
+ * @param {(Array<!window.PaymentDetailsModifier>|undefined)} modifiers
+ */
+function convertPaymentDetailsModifiers(modifiers) {
+  if (!modifiers || !(modifiers instanceof Array)) {
+    return;
+  }
+
+  for (let i = 0; i < modifiers.length; i++) {
+    if (modifiers[i].total) {
+      convertPaymentCurrencyAmount(modifiers[i].total.amount);
+    }
+    convertDisplayItems(modifiers[i].additionalDisplayItems);
+  }
+}
+
+module.exports = (window, navigator) => {
+  if (!window.PaymentRequest) {
+    return;
+  }
+
+  if (/CriOS\/(62|63)/.test(navigator.userAgent)) {
+    let originalPaymentRequest = window.PaymentRequest;
+
+    window.PaymentRequest = function(methodData, details, optOptions) {
+      convertPaymentDetails(details);
+      originalPaymentRequest.call(this, methodData, details, optOptions);
+    };
+
+    window.PaymentRequest.prototype.__proto__ =
+        originalPaymentRequest.prototype;
+
+    /* eslint-disable no-undef */
+    let originalUpdateWith = __gCrWeb['paymentRequestManager'].updateWith;
+
+    __gCrWeb['paymentRequestManager'].updateWith = function(detailsOrPromise) {
+      // if |detailsOrPromise| is not an instance of a Promise, wrap it in a
+      // Promise that fulfills with |detailsOrPromise|.
+      if (!detailsOrPromise || !(detailsOrPromise.then instanceof Function) ||
+          !(detailsOrPromise.catch instanceof Function)) {
+        detailsOrPromise = Promise.resolve(detailsOrPromise);
+      }
+
+      let self = this;
```

Comment:
> `let` is not fully supported in iOS9. Don't use it in code that runs on Chrome for iOS, until support for iOS9 is dropped.

https://chromium.googlesource.com/chromium/src/+/0b97e14a74da4b5aa9f5e1d8997c20df0e5dc875/docs/es6_chromium.md#let-Block_Scoped-Variables

Reply:
done.

Reply:
let and const seem to get converted to var in the final payment-shim.js